PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/View Controllers/Detailed ViewControllers/stockDetailedViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  stockDetailedViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Kary Martinez on 9/5/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class stockDetailedViewController: UIViewController {
+    
+    @IBOutlet weak var downLabel: UILabel!
+    @IBOutlet weak var UPLabel: UILabel!
+    
+    @IBOutlet weak var stockImage: UIImageView!
+    
+    @IBOutlet weak var dateLabel: UILabel!
+    
+    var currentStockinfo: Stock!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        dateLabel.text = currentStockinfo.date
+        Colors()
+        thumbsUpOrThumbsDownImage()
+        UPLabel.text = "Open: \(currentStockinfo.uOpen)"
+        downLabel.text = "Close: \(currentStockinfo.uClose)"
+        
+    }
+    private func thumbsUpOrThumbsDownImage() {
```

Comment:
Great job making this a private func. Butttt, This logic should really be done in the model. Make an instance function in the Stocks model that compares the stored properties open and close. You would want to return something, you call that func in the viewDidLoad and based on the results you update the view color and image appropriately.